recipe/meta.yaml

```diff
@@ -25,18 +25,18 @@ requirements:
   run:
     - python >=3.5.3
     - PyYAML
-    - google-cloud-storage >=1.13.0
-    - python-kubernetes >=8.0.0,<12.0.0
+    - google-cloud-storage >=1.13
+    - python-kubernetes >=8,<12
     - google-auth >=1.6.1
-    - requests-toolbelt >=0.8.0
     - cloudpickle
-    - kfp-server-api >=0.2.5,<2.0.0
+    - kfp-server-api >=1.1.12,<=2.0.0
     - jsonschema >=3.0.1
     - tabulate
     - click
     - Deprecated
     - strip-hints
-    - docstring_parser
+    - docstring-parser >=0.7.3
+    - kfp-pipeline-spec >=0.1.0,<0.2.0
```

Comment:
maybe one hacky way to work around the missing `kfp-pipeline-spec` feedstock, could be to remove it from the `requirements` here and instead amend the build section to `pip install` it just prior to installing `kfp` itself 

```YAML
build:
  noarch: python
  number: 0
  script:
    - {{ PYTHON }} -m pip install kfp-pipeline-spec==0.1.4 -vv
    - {{ PYTHON }} -m pip install . -vv
  entry_points:
    - dsl-compile = kfp.compiler.main:main
    - kfp=kfp.__main__:main
```

Reply:
@ckadner I don't mind using this workaround until "fp-pipeline-spec" is on conda.

I have added as a maintainer for this feedstock so you should be able to make the changes.

Reply:
Thank you @maximerihouey for adding me as a maintainer. My hack failed. I will try to spend some time on a new recipe for `kfp-pipeline-spec`